### PR TITLE
Feature/documention update

### PR DESCRIPTION
**Pull request by garyluu**
Fixes 4 issues.

Issue #11 should be fixed as I do not see any more broken links.  Removed document line of test to change because no change should be needed.

Issue #10 write-api-service now mentions how to generate a Quay.io token.

Issue #9 Properties/configuration files were already mentioned in the respective client and service but now it's made more obvious that it's needed before running tests.

Issue #8 Badges added for develop branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
+![Build Status](https://travis-ci.org/dockstore/write_api_service.svg?branch=develop)
```
Comment (denis-yuen) [inline README.md:1]: This still requires more explanation. 
Imagine yourself as a developer with a passing knowledge of Dockstore that comes across this. 
Explain, what this tool is used for, who needs to use this tool, what this tool replaces, etc.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 The Write-API allows developer to automatically create GitHub and Quay repositories for any tool encapsulated in Docker and described with a CWL descriptor on behalf of the user.
 It implements the proposed GA4GH [Write API](https://github.com/ga4gh/tool-registry-schemas/blob/feature/write_api_presentation/src/main/resources/swagger/ga4gh-tool-discovery.yaml)
 
 This contains two parts:
 - The Write API web service that handles creation of GitHub and Quay repositories
 - The client that interacts with the web service and handles publishing of tools to Dockstore.
 
+To build the service and the client use:
+```
+mvn clean install -DskipTests
```
Comment (denis-yuen) [inline README.md]: The tests presumably create and tear down repos. Note which repos those are and how to configure them.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,16 @@
...
+```
+Running tests will require configuration/properties files for both client and service.  Do not try to run "mvn clean install" without setting up those files first.  Additionally, both client and service tests will require Quay.io and GitHub tokens.
```
Comment (denis-yuen) [inline README.md]: Let's have something clear and step-by-step here. I don't care whether it is in each project, but the documentation currently isn't clear at all how to get the tests running.

File: write-api-client/README.md [hunk 4]
```diff
@@ -17,6 +17,9 @@ repo=test-repository
 write-api-url=http://localhost:8082/api/ga4gh/v1
 ```
 
+After building, the CLI's jar file can be found under at the directory:  write-api/write-api-client/target/
+
```
Comment (denis-yuen) [inline write-api-client/README.md]: Let's add some scenarios. 
Add a few example CLI invocations and explain the result.

**Review by denis-yuen — CHANGES_REQUESTED** [review]
Needs a lot more explanation and examples

File: write-api-client/README.md [hunk 5]
```diff
@@ -17,6 +17,9 @@ repo=test-repository
 write-api-url=http://localhost:8082/api/ga4gh/v1
```
Comment (denis-yuen) [inline write-api-client/README.md]: Document each property, in particular, `repo` is a bit puzzling

**Comment by garyluu** [comment]
Made a few documentation changes

**Pull request closed without merging**